packages/shorebird_cli/lib/src/cache.dart

```diff
@@ -104,15 +104,15 @@ class Cache {
 
   String get storageBucket => 'download.shorebird.dev';
 
-  void clear() {
+  Future<void> clear() async {
     final cacheDir = shorebirdCacheDirectory;
     if (cacheDir.existsSync()) {
-      cacheDir.deleteSync(recursive: true);
+      await cacheDir.delete(recursive: true);
```

Comment:
We can do them in parallel in this case right?

Reply:
Good call, done